charts/vela-core/templates/definitions/terraform-alibaba-rds.yaml

```diff
@@ -16,10 +16,11 @@ spec:
     terraform:
       configuration: |
         module "rds" {
-          source = "terraform-alicloud-modules/rds/alicloud"
+          source = "github.com/zzxwill/terraform-alicloud-rds"
```

Comment:
@wonderflow I will revert this PR once https://github.com/terraform-alicloud-modules/terraform-alicloud-rds/pull/23 is merged.